.circleci/docker/common/install_vision.sh

```diff
@@ -47,11 +47,16 @@ install_centos() {
 }
 
 # Install base packages depending on the base OS
-if [ -f /etc/lsb-release ]; then
-  install_ubuntu
-elif [ -f /etc/os-release ]; then
-  install_centos
-else
-  echo "Unable to determine OS..."
-  exit 1
-fi
+ID=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
+case "$ID" in
+  ubuntu)
+    install_ubuntu
+    ;;
+  centos)
+    install_centos
+    ;;
+  *)
+    echo "Unable to determine OS..."
+    exit 1
+    ;;
+esac
```

Comment:
Hmm, looks like we have an identical tail in `install_vision.sh`, `install_protobuf.sh`, etc, perhaps this could be refactored into a common function?

Reply:
Yes, I didn't like the copy/paste coding here.  But I didn't see any mechanism such as a commonly included or sourced script that could be shared between all these `common/install_foo.sh` scripts, because we use docker `ADD common/install_foo.sh` and this would require adding additional files, etc.